src/proc_image_processing/cpu/server/filter_chain.cc

```diff
@@ -118,6 +121,8 @@ namespace proc_image_processing {
                 ROS_ERROR("[FILTERCHAIN %s ], Error in image processing: %s", name_.c_str(), e.what());
             };
         }
+        // Uncomment with the timer to see filterchain performance
+        //ROS_INFO("Filterchain exec time: %f", timer.getExecutionTime());
```

Comment:
Should I let this ?

Reply:
Non (ça consomme en théorie des ressources si on sen sert pas). Mais ça pourrait être bien d'ajouter un debug pour chaque filter qu'on peut activer quand on test